src/common/components/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -0,0 +1,20 @@
+import { Tooltip, Typography } from "antd";
+import React, { ReactElement } from "react";
+import { useSelector } from "react-redux";
+import { selectCurrentPage } from "../../../app/mainSelectors";
+import { PageType } from "../../../app/mainSlice.types";
+import { selectCurrentPageObject } from "../../../features/pageObjects/pageObjectSelectors";
+
+export const Breadcrumbs = (): ReactElement | null => {
+  const currentPage = useSelector(selectCurrentPage);
+  const pageObject = useSelector(selectCurrentPageObject);
+
+  return currentPage.page === PageType.LocatorsList ? (
+    <div className="jdn__breadcrumb">
```

Comment:
could you, please, rename this style to `jdn__breadcrumbs` to follow component's name?